app/views/movies/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= @movie.title %>
+<%= @movie.creation_year %>
+<%= @movie.genre %>
+<%= @movie.actor_age_avg %>
+
+<% @movie.actors_sorted.each do |actor| %>
+  <div id="actor-<%= actor.id %>">
+    <%= actor.name %>
+    <%= actor.age %>
+  </div>
+<% end %>
```

Comment:
While as backend devs we are always going to be tasked with writing HTML, since we are creating our own views, you will need to use HTML in your logic. Right now your erb tags are just hanging out all willy-nilly. They should all be enclosed in some sort of HTML tags. For example:

```html.erb
<h1><%= @movie.title %></h1>
<h2><%= @movie.creation_year %><h2>
<h2><%= @movie.genre %><h2>
<p>Average actor age: <%= @movie.actor_age_avg %></p>
```